### DISCUSSION TITLE
Gaussian SCF error in DP-GEN fp process

### DISCUSSION THREAD

**Discussion by ghost** [deepmodeling/dpgen#1408] [Q&A] [answered]
When running Gaussian16, an SCF error occurs due to incorrect structure. Since the error does not result in a normal completion, a completion flag file is not generated and the step cannot proceed any further.
Is it possible to ignore the structure that causes the SCF error? Or do I have to configure the SCF settings on the Gaussian16 side and use unreliable calculated values?

parameters:

type_map:
  - Na
init_data_prefix: ''
init_data_sys:
  - /home/sekinelabo/Documents/DeepMD-Sodium/dpgen/data/Na7/
sys_format: deepmd/npy
sys_configs:
  - - /home/sekinelabo/Documents/DeepMD-Sodium/dpgen/data/Na7/
numb_models: 4
default_training_param:
  model:
    type_map:
    - Na
    data_stat_nbatch: 1
    type: standard
    descriptor:
      type: se_e2_a
      sel:
      - 60
      rcut: 6.0
      rcut_smth: 1.8
      neuron:
      - 25
      - 50
      - 100
      axis_neuron: 8
      resnet_dt: false
      precision: float64
      seed: 1
    fitting_net:
      type: ener
      neuron:
      - 120
      - 120
      - 120
      precision: float64
      resnet_dt: true
      seed: 1
  learning_rate:
    type: exp
    start_lr: 0.001
    stop_lr: 3.0e-08
    decay_steps: 5000
  loss:
    type: ener
    start_pref_e: 0.02
    limit_pref_e: 1
    start_pref_f: 1000
    limit_pref_f: 1
    start_pref_v: 0
    limit_pref_v: 0
  training:
    numb_steps: 1000000
    seed: 1
    disp_file: lcurve.out
    disp_freq: 100
    save_freq: 1000
    save_ckpt: model.ckpt
    disp_training: true
    time_training: true
    profiling: false
    profiling_file: timeline.json
fp_task_max: 20
fp_task_min: 1
model_devi_engine: lammps
model_devi_jobs:
  - sys_idx:
      - 0
    temps:
      - 300
    press:
      - 1
    nsteps: 1000
    ensemble: nvt
    trj_freq: 10
    _idx: "00"
  - sys_idx:
      - 0
    temps:
      - 300
    press:
      - 1
    nsteps: 3000
    ensemble: nvt
    trj_freq: 10
    _idx: "01"
model_devi_dt: 0.002
model_devi_skip: 0
model_devi_f_trust_lo: 0.05
model_devi_f_trust_hi: 0.15
fp_style: gaussian
use_clusters: false
fp_params:
  keywords: P B3LYP/6-311+g(d) force
  nproc: 14
  charge: 0

train:
  command: dp
  machine:
    batch_type: Shell
    context_type: local
    local_root: ./
    remote_root: /home/sekinelabo/Documents/DeepMD-Sodium/dpgen/
  resources:
    gpu_per_node: 1
    group_size: 14
    batch_type: Shell
model_devi:
  command: lmp_mpi
  machine:
    batch_type: Shell
    local_root: ./
    remote_root: /home/sekinelabo/Documents/DeepMD-Sodium/dpgen/
    context_type: local
  resources:
    gpu_per_node: 1
    group_size: 14
    batch_type: Shell
fp:
  command: g16 < input > output
  machine:
    batch_type: Shell
    local_root: ./
    remote_root: /home/sekinelabo/Documents/DeepMD-Sodium/dpgen/
    context_type: local
  resources:
    gpu_per_node: 1
    cpu_per_node: 14
    group_size: 1
    batch_type: Shell

**Comment by njzjz** [comment, marked as answer]
Just appending `||:` to a bash command will make the exit code zero, as shown below.

https://github.com/deepmodeling/dpgen/blob/042ea32e3dd69191f00bd2fe0151047b591ca92f/examples/machine/DeePMD-kit-1.x/machine-pbs-gaussian.json#L86

**Reply by ghost** [reply]
Thank you for your replying! That's solved for your answer!
